store/src/java/com/zimbra/cs/dav/carddav/CustomXmlDetector.java

```diff
@@ -0,0 +1,49 @@
+package com.zimbra.cs.dav.carddav;
```

Comment:
@mamtadahiwal have you tried verifying the content type from Tika mime type detection using https://github.com/Zimbra/zm-mailbox/blob/develop/store-conf/conf/custom-mimetypes.xml

Reply:
yes , it is not detecting using custom-mimetypes.xml , I have tried changing dependency version as well as i have try with adding some xml tag and changing in existing file also